spec/services/passporting/ioj_passporter_spec.rb

```diff
@@ -118,4 +120,50 @@
       end
     end
   end
+
+  describe '#offence_passported?' do
+    before do
+      allow(
+        FeatureFlags.offence_ioj_passport
+      ).to receive(:enabled?).and_return(feat_enabled)
+    end
+
+    context 'feature flag is disabled' do
+      let(:feat_enabled) { false }
+
+      it { expect(subject.offence_passported?).to be(false) }
+    end
+
+    # rubocop:disable RSpec/MultipleMemoizedHelpers
+    context 'feature flag is enabled' do
+      let(:feat_enabled) { true }
+
+      let(:passported_charge) { Charge.new(offence_name: 'Attempt robbery') }
+      let(:non_passported_charge) { Charge.new(offence_name: 'Affray') }
+      let(:non_listed_charge) { Charge.new(offence_name: 'This is a test offence') }
+
+      before do
+        allow(passported_charge.offence).to receive(:ioj_passport).and_return(true)
+      end
+
+      context 'when there is at least one passported offence' do
+        let(:charges) { [non_passported_charge, passported_charge] }
+
+        it { expect(subject.offence_passported?).to be(true) }
+      end
+
+      context 'when there is at least one passported offence but also non listed offences' do
+        let(:charges) { [non_listed_charge, non_passported_charge, passported_charge] }
+
+        it { expect(subject.offence_passported?).to be(true) }
+      end
+
+      context 'when there are no passported offences' do
+        let(:charges) { [non_listed_charge, non_passported_charge] }
+
+        it { expect(subject.offence_passported?).to be(false) }
+      end
+    end
+    # rubocop:enable RSpec/MultipleMemoizedHelpers
+  end
```

Comment:
Any use in adding a spec to test a combination of passported on age and passported on offence?

Reply:
Yep, good point. I'll add some coverage for that 👍